webhook/README.md

```diff
@@ -0,0 +1,69 @@
+Webhook
+---------------
+
+Module to receive [global webhooks](https://es.wikipedia.org/wiki/Webhook) events.
+This module invoke methods to process webhook events.
+You will need create a new module to add your logic to process the events with methods called:
+`def run_CONSUMER_EVENT*`
```

Comment:
Why the star?
Maybe say: "The webhook events will be processed by your custom methods, that should be prefixed with `run_`"